### ISSUE TITLE
halias loops when existing host aliases aren't accessible by the effective user

### ISSUE THREAD

**Issue by fscheiner** [fr4nk5ch31n3r/gtransfer#37]
The issue title already tells everything.

tl;dr

In situations where an administrator creates host aliases, but uses a restrictive umask (e.g. `0077`), the resulting files are usually not readable by ordinary users. If a user then wants to dealias a given host alias, `halias` will try forever to get the dealiased value (as the file name itself can be read by the user if the containing dir can be accessed). The bugs are in [`lib/gtransfer/alias.bashlib`] in the _if clauses_ on lines [177] and [215]. There needs to be an _else_ part, that leaves the _while loop_ if the alias content is empty or the alias cannot be accessed.

[`lib/gtransfer/alias.bashlib`]: https://github.com/fr4nk5ch31n3r/gtransfer/blob/9a2930c7e427aeb09f615bd3acdeff6e9d632108/lib/gtransfer/alias.bashlib

[177]: https://github.com/fr4nk5ch31n3r/gtransfer/blob/9a2930c7e427aeb09f615bd3acdeff6e9d632108/lib/gtransfer/alias.bashlib#L177

[215]: https://github.com/fr4nk5ch31n3r/gtransfer/blob/9a2930c7e427aeb09f615bd3acdeff6e9d632108/lib/gtransfer/alias.bashlib#L215
